website/blog/2019-10-31-oclif-node-updates.md

```diff
@@ -0,0 +1,70 @@
+---
+author: Chad Carbert
+title: oclif's Current Node Support
+---
+
+To maintain a healthy project trajectory, oclif follows and supports [Node Active LTS release](https://nodejs.org/en/about/releases/), currently Node 10 & Node 12. This means ensuring that oclif continues to play nice with coming Active LTS Node versions and other packages in the ecosystem. Moving forward also means leaving older versions behind. Starting in 2020, Node will stop maintaining version 8 (https://github.com/nodejs/Release#release-schedule) and it is our intent at that time to also follow suit. Let’s take a look at a few ways we will be supporting these changes.
+
+## CI Environments
+
+CLIs created with the oclif cli going forward will be generated with a CircleCI configuration with Node 10 & 12 and an Appveyor configuration using Node 10. We have also added Node latest to CircleCi to be an early warning detection against coming Node changes (Node latest is managed by CircleCI).
+
+We have already updated every oclif repo's CI configs to reflect this.
+
+If your existing CLI uses either Appveyor or CircleCI you can update your config files also, like so:
+
+### .circleci/config.yml
+
+Your CircleCI config should contain a `node-latest` job, aliased as `test`. From this, there should be two extensions of this job for the Active LTS Node versions, Node 10 and Node 12.
+
+```
+  node-10:
+    <<: *node-latest
+    docker:
+      - image: node:10
+  node-12:
+    <<: *node-latest
+    docker:
+      - image: node:12
+```
+
+Notice that these declarations only change the Docker Node images used to run that job.
+
+Additionally, the jobs listed within workflows must also be updated to reflect our changes in configuration:
+
+```
+    jobs:
+      - node-latest
+      - node-10
+      - node-12
+```
+
+### appveyor.yml
+
+For appveyor we are currently only testing the oldest Active LTS Node version, Node 10. Update the `nodejs_version` proppert in your appveyor.yml file to reflect this.
```

Comment:
is something preventing us from testing node 12?

Reply:
Just slowness of appveyor